src/LibraryManager.Vsix/Resources/Text.resx

```diff
@@ -235,4 +235,14 @@ Do you want to continue?</value>
   <data name="Install" xml:space="preserve">
     <value>_Install</value>
   </data>
+  <data name="SuggestedAction_Update_Prerelease" xml:space="preserve">
```

Comment:
I'm guessing this is not for preview 4 as we need another loc pass.

Reply:
That's true.  I can hold this one back for now.